### DISCUSSION TITLE
News

### DISCUSSION THREAD

**Discussion by shervin8066** [community/community#30774] [Other Feature Feedback, Questions, & Ideas]
Hello, please add Farsi language to the news section of Kiwi browser's home page. In Google Chrome browser, there is Farsi language in the news section and it is surprising to me why Kiwi browser has all languages ​​and not only Farsi.

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/kiwibrowser/src? In that case you should open an issue *in that repository* so the developers see it. :slightly_smiling_face:
